app/src/main/java/chat/rocket/android/settings/presentation/settingPresenter.kt

```diff
@@ -27,25 +28,22 @@ class settingPresenter @Inject constructor(
 
 
 
-
+// this method is written to execute download data call
```

Comment:
What I want to mean was: You don't need to leave the previous comment here (copy and paste error?) since there is no relation with your code.

The comment is ok now, but from the signature of the function it is meaningful enough. I mean: Not need to do that, but ok.